hardened-chromium-source.spec

```diff
@@ -0,0 +1,59 @@
+%global numjobs %{_smp_build_ncpus}
+
+Name:	 hardened-chromium-source
+Version: 130.0.6723.91
+Release: clean
+Summary: Cleaned chromium source.
+Url:     http://www.chromium.org/Home
+License: BSD-3-Clause
+
+BuildRequires: git
+BuildRequires: python3
+
+%description
+Vanilla, cleaned, source of chromium. Used by hardened-chromium.
+
+%build
+# obtain cleaning utilities
+git clone https://github.com/secureblue/hardened-chromium-source-cache.git
+cp hardened-chromium-source-cache/* .
+
+# obtain depot tools for obtaining source
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
```

Comment:
are you using the "custom build" build method? if so we could move some of this to the src script step. Currently it seems like we're not doing anything in the src step for this?